index.d.ts

```diff
@@ -1,3 +1,7 @@
+declare interface FigureSet {
```

Comment:
Needs a doc comment on what it is and how it's useful. Preferably with an example.

This needs to be documented in the readme too.